source/drivers.c

```diff
@@ -10,14 +12,26 @@ static lv_color_t g_buffer[LV_HOR_RES_MAX * LV_VER_RES_MAX];
 static touchPosition g_touch_pos;
 
 static lv_group_t *g_keypad_group;
-
+// sixaxis handles
+static u32 handles[4];
+// we actually only need to keep track of a specific set of data so we dont need all sixaxis values
+static HidVector g_gyro_center;
+// boundries for translating gyro directly to screen space
+// smaller values for smaller tvs/smaller motions
+static lv_obj_t * g_pointer_canvas;
+static lv_img_dsc_t g_pointer_img;
+static lv_color_t g_pointer_buff[LV_CANVAS_BUF_SIZE_TRUE_COLOR_ALPHA(96, 96)];
+static lv_obj_t * g_pointer_fake_canvas;
+static float g_pointer_screen_magic = 0.7071f; // this is a repeating number that describes the top right of a square inside a unit circle whose sides are parallel to the x-y axis'
 static void flush_cb(lv_disp_drv_t *drv, const lv_area_t *area, lv_color_t *color_p) {
     u32 stride;
     lv_color_t *fb = (lv_color_t *) framebufferBegin(&g_framebuffer, &stride);
 
     for (int y = area->y1; y <= area->y2; y++) {
         for (int x = area->x1; x <= area->x2; x++) {
-            fb[y * stride / sizeof(lv_color_t) + x] = *color_p;
+            u32 pos = y * stride / sizeof(lv_color_t) + x;
```

Comment:
This code is from an old commit, please change so that it's in line with current code